### PR TITLE
Export PDF of Users with  QR Code IDs

### PR DESCRIPTION
**Pull request by enjeyw**
**Describe the Pull Request**

Allows for an export of a PDF of users with QR code IDs - useful for attaching user data after sign up by referencing a printed form

**Todo**

- [ ] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [ ] Request review from relevant @person or team
- [ ] QA your pull request. This includes running the app, login, testing changes etc.
- [ ] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: app/client/components/management/exportManager.jsx [hunk 1]
```diff
@@ -94,6 +98,18 @@ class ExportManager extends React.Component {
   }
 
   render() {
+    let transferToggle = (
```
Comment (enjeyw) [inline app/client/components/management/exportManager.jsx:101]: shifted from below to a var so we can optionally render it for spreadsheet export only

File: app/client/components/management/exportManager.jsx [hunk 2]
```diff
@@ -133,48 +149,43 @@ class ExportManager extends React.Component {
...
-                fontWeight: "400",
-                margin: "0 1em",
-                lineHeight: "25px",
-                height: "25px"
-              }}
-              name="account_type"
-              value={this.state.account_type}
-              onChange={this.handleChange}
+            <p style={{ margin: "0px 1em 0px 0px" }}>Export format:</p>
+            <Select
```
Comment (enjeyw) [inline app/client/components/management/exportManager.jsx:153]: 🐜 🥳

File: app/server/utils/export.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    def pdf_save_meth(path):
+        HTML(string=html).write_pdf(path)
+
+    return _export_via_s3(pdf_save_meth, filename, email)
+
+
+def export_workbook_via_s3(wb, filename, email=None):
+
+    def wb_save_method(path):
```
Comment (enjeyw) [inline app/server/utils/export.py:36]: dat function injection

File: app/Dockerfile [hunk 4]
```diff
@@ -10,6 +10,8 @@ RUN cd / && pip install -r requirements.txt && pip install -r test_requirements.
 
 RUN apt remove --purge -y gcc g++ libffi-dev libstdc++-8-dev python3-dev musl-dev libssl-dev
 
+RUN apt -y install libcairo2 libpango-1.0-0 libpangocairo-1.0-0 libgdk-pixbuf2.0-0 libffi-dev shared-mime-info
```
Comment (tristanhcole) [inline app/Dockerfile:13]: Locally, I needed to run `brew install cairo && brew install pango`

**Review by tristanhcole — APPROVED** [review]
LGTM!

**Pull request merged**